lib/auth_page.dart

```diff
@@ -66,6 +83,56 @@ Stream<int> _signin(BuildContext context) async* {
   }
 }
 
+void _facebookSignout() {
```

Comment:
Is this used by anything?

Reply:
Oh I see, the page still has a sign out button -> with the Options Page sign out option in the main branch now, has this has become superfluous?

Reply:
Wasn't there a comment that it might be preferred to signout each google + facebook authenication provider before firebase authentication provider?

Reply:
I was trying to ask was the function on that line being used (_facebookSignout).  I was assuming the sign out feature had been taken out and the function might have accidentally been left in.  Then I realised the sign out feature hadn't been taken out.  Sorry if I wasn't clear.  

We don't want to have a sign out feature in the auth page right?  My understanding was it was just to make testing the facebook signin easier?  So now that there is a sign out in the Options page we should just remove all the sign out stuff from the auth page... or am I missing something? (entirely likely!) 

I agree we should sign out of the relevant providers (including facebook) when sign out occurs though.